src/main/java/com/solvd/carina_testing/TodayPage.java

```diff
@@ -0,0 +1,81 @@
+package com.solvd.carina_testing;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class TodayPage extends AbstractPage {
+
+    @FindBy(xpath = "//*[@content-desc='Quick add']")
+    private ExtendedWebElement addTaskButton;
+
+    @FindBy(xpath = "(//*[@text='Today'])[1]")
+    private ExtendedWebElement titleToday;
+
+    @FindBy(xpath = "//*[@resource-id='android:id/message']")
+    private ExtendedWebElement inputTitleTaskLine;
+
+    @FindBy(xpath = "//*[@content-desc='Add']")
+    private ExtendedWebElement airplaneButton;
+
+    @FindBy(xpath = "(//*[@resource-id='com.todoist:id/item'])")
+    private List<TaskCard> taskCards;
+
+    @FindBy(xpath = "//*[@resource-id='com.todoist:id/compose_navigation_holder']/*/*/*")
+    private Footer footer;
+
+    @FindBy(xpath = "(//*[@resource-id='com.todoist:id/text'])[2]")
+    private ExtendedWebElement hideKeyboard;
+
+    @FindBy(xpath = "//*[@resource-id='com.todoist:id/compose_navigation_holder']/*/*/*/*[4]/*[1]")
```

Comment:
this button is located in footer, remove it and methods related to it from this page